CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
 
 namespace CompanyEmployees.Core.Domain.Entities
```

Comment:
The `Salary` property should be nullable (`decimal?`) if it's not required for all employees, to reflect optional data and avoid default zero values being misinterpreted as valid salaries.

```suggestion
public decimal? Salary { get; set; }
```